test_themes/tests/test_new_page_templates.py

```diff
@@ -190,6 +190,13 @@ def check(theme_name, website):
                                 conflict.difference_update(white_list)
                                 if len(conflict) > 1:
                                     errors.append("Using %r, view %r contains conflicting classes: %r in %r (according to pattern %r)" % (theme_name, view.key, conflict, classes, conflicting_classes_re.pattern))
+                        for el in html_tree.xpath('//*[@style]'):
+                            styles = el.attrib['style'].split(';')
+                            properties = list(map(lambda x: x.split(':')[0].strip(),
+                                filter(lambda x: x, styles)
+                            ))
```

Comment:
This is very hard to read
Maybe we should extract some variables?
I also like to name the loop variable with a clear name. In this case, I would rename `x` with `style`
`filter(lambda x: x, styles)` is a little tricky to understand, maybe worth a comment?

Reply:
I would also consider naming this variable `property_names` instead of `properties`, but your choice on this

Reply:
Added an intermediary variable `non_empty_styles` instead of a comment.